### PR TITLE
pythonPackages.pytest_3: drop

### PR DESCRIPTION
**Pull request by marsam**
###### Motivation for this change
See: https://github.com/NixOS/nixpkgs/pull/64445#issuecomment-510009002

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @FRidh

### CODE REVIEW

File: pkgs/development/python-modules/fastpair/default.nix [hunk 1]
```diff
@@ -11,14 +11,17 @@ buildPythonPackage {
...
 
-  checkInputs = [ pytest_3 ];
+  checkInputs = [ pytest ];
 
   propagatedBuildInputs = [
     scipy
   ];
 
+  # Does not support pytest 4 https://github.com/carsonfarmer/fastpair/issues/14
+  doCheck = false;
```
Comment (risicle) [inline pkgs/development/python-modules/fastpair/default.nix]: :(

**Comment by risicle** [comment]
Sigh. WFM on macos 10.13.

I guess.

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS (1 broken on master)
diff LGTM
most binaries seem to work

```
[22 built (1 failed), 64 copied (129.0 MiB), 41.5 MiB DL]
error: build of '/nix/store/c72p5sradqfdlz9q4gxdc39yys1gwby0-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/66376
1 package failed to build:
python37Packages.sunpy

18 package were build:
bonfire datasette python27Packages.click-default-group python27Packages.fastpair python27Packages.modeled python27Packages.moretools python27Packages.robotframework-tools python27Packages.zeep python27Packages.zetup python37Packages.click-default-group python37Packages.fastpair python37Packages.matchpy python37Packages.modeled python37Packages.moretools python37Packages.pysonos python37Packages.robotframework-tools python37Packages.zeep python37Packages.zetup
```

**Review by jonringer — COMMENTED** [review]
not related to your changes, but can you change bonfire and datasette to using python2, i get this error when trying to run them:

```
[nix-shell:~/.cache/nix-review/pr-66376-1]$ ./results/bonfire/bin/bonfire
Error in sitecustomize; set PYTHONVERBOSE for traceback:
ImportError: This package should not be accessible on Python 3. Either you are trying to run from the python-future src folder or your installation of python-future is corrupted.
...
```
```
[nix-shell:~/.cache/nix-review/pr-66376-1]$ ./results/bonfire/bin/bonfire
Error in sitecustomize; set PYTHONVERBOSE for traceback:
ImportError: This package should not be accessible on Python 3. Either you are trying to run from the python-future src folder or your installation of python-future is corrupted.
...
```

**Comment by FRidh** [comment]
Thanks!

**Pull request merged**